thorlcr/activities/diskread/thdiskread.cpp

```diff
@@ -31,9 +31,11 @@ class CDiskReadMasterVF : public CDiskReadMasterBase
 {
 public:
     CDiskReadMasterVF(CMasterGraphElement *info) : CDiskReadMasterBase(info) { }
-    virtual void validateFile(IDistributedFile *file)
+    virtual void validateFile(IDistributedFile *file) override
     {
         IHThorDiskReadBaseArg *helper = (IHThorDiskReadBaseArg *)queryHelper();
+        if (!isFileFlat(file))
```

Comment:
I thought this test was only going to be for variable size files - since they are the only files that can crash when reading mismatching files.  It is reasonable to read a fixed with csv file as binary for example.

Reply:
I will change to only check in variable case.